src/value/owned.rs

```diff
@@ -610,89 +596,100 @@ impl IonElement for Element {
         }
     }
 
-    fn annotations(&self) -> Self::AnnotationsIterator<'_> {
+    pub fn annotations(&self) -> SymbolsIterator<'_> {
         SymbolsIterator::new(&self.annotations)
     }
 
-    fn with_annotations<I: IntoIterator<Item = Self::SymbolToken>>(self, annotations: I) -> Self {
-        Element::new(annotations.into_iter().collect(), self.value)
+    pub fn with_annotations<S: Into<Symbol>, I: IntoIterator<Item = S>>(
+        self,
+        annotations: I,
+    ) -> Self {
+        let annotations: Vec<Symbol> = annotations.into_iter().map(|i| i.into()).collect();
+        Element::new(annotations, self.value)
     }
 
-    fn has_annotation(&self, annotation: &str) -> bool {
+    pub fn has_annotation(&self, annotation: &str) -> bool {
         self.annotations
             .iter()
             .any(|a| a.text() == Some(annotation))
     }
 
-    fn is_null(&self) -> bool {
+    pub fn is_null(&self) -> bool {
         matches!(&self.value, Value::Null(_))
     }
 
-    fn as_integer(&self) -> Option<&Integer> {
+    pub fn as_integer(&self) -> Option<&Integer> {
         match &self.value {
             Value::Integer(i) => Some(i),
             _ => None,
         }
     }
 
-    fn as_f64(&self) -> Option<f64> {
+    pub fn as_float(&self) -> Option<f64> {
```

Comment:
🗺️ I renamed this to reflect the Ion type (`float`) rather than the Rust type (`f64`) as we do in the other methods.